### ISSUE TITLE
Migration failure on Oracle Standard Edition During CREATE INDEX Step

### ISSUE THREAD

**Issue by KevinDougan-SmileCDR** [hapifhir/hapi-fhir#3924]
Problem Statement
Migration 2022.05.R01 to 2022.08.R01 fails on Oracle Standard Edition during CREATE INDEX step

Steps to Reproduce
Install 2022.05.R01 using Oracle Standard Edition
Upgrade to 2022.08.R01
Observed Results
```
Caused by: org.springframework.jdbc.UncategorizedSQLException: PreparedStatementCallback; uncategorized SQLException for SQL [create index IDX_EMPI_MATCH_TGT_VER on MPI_LINK(MATCH_RESULT, TARGET_PID, VERSION) ONLINE DEFERRED INVALIDATION]; SQL state [67000]; error code [439]; ORA-00439: feature not enabled: Online Index Build
; nested exception is java.sql.SQLException: ORA-00439: feature not enabled: Online Index Build
```

Expected Results
No errors

Environment (please complete the following information):
hapi-fhir 6.1.0 upgrade
Oracle DB Server Standard Edition

Oracle DB Server Standard Edition doesn't support ONLINE option in the CREATE INDEX command

**Comment by karthikchundi-commits** [comment]
I dug into this. The relevant code is in `MetadataSource.getEdition()` (hapi-fhir-sql-migrate):

```java
result = theConnectionProperties.newJdbcTemplate()
        .queryForObject("SELECT BANNER FROM v$version WHERE banner LIKE 'Oracle%'", String.class);
```

and then in `isOnlineIndexSupported()`:

```java
return oracleEdition == null || oracleEdition.contains("Enterprise");
```

So `.online(true)` on `AddIndexTask` should already be blocked on Standard Edition via this check, unless the edition string isn't matching the way it's expected to.

One thing worth checking: Oracle's `v$version` view added a `BANNER_LEGACY` column starting in Database 18c, specifically to preserve the pre-18c single-line banner format - which implies the plain `BANNER` column's content/format changed starting in that release. This code only queries `BANNER`, not `BANNER_LEGACY`. If `BANNER`'s post-18c format doesn't reliably include "Enterprise"/edition text the same way it used to, `getEdition()` could be returning something that doesn't match `.contains("Enterprise")` correctly, which would explain the `ONLINE` clause still getting added on Standard Edition on newer Oracle versions.

If anyone can reproduce this on a Standard Edition instance, it'd help to confirm with:
```sql
SELECT BANNER, BANNER_LEGACY FROM v$version WHERE banner LIKE 'Oracle%';
```
That would confirm whether this is actually a `BANNER` vs `BANNER_LEGACY` issue, or something else in the edition detection.
